### DISCUSSION TITLE
Limit wurde nach Update auf v24.1.4 auf 0 gesetzt

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#1621] [General]
Hi,

war das nur Zufall jetzt bei mir nach dem OTA Update oder ist das bei noch jemandem aufgetreten?

**Comment by MichlW** [comment]
Bei mir ist alles gut und es steht weiterhin auf 100%.
(2x HMS-1600)

**Comment by 3aau** [comment]
bei mir ist das limit auch auf 0 gegangen

**Comment by chaos0651** [comment]
Hallo und einen guten Abend 
Ja bei mir auch nach neustart auf 0% geht aber meist nach einer weile wieder auf den eingestellten wert zurück 
aber die leistung des WR wird nicht verändert und bleibt auf seinem eingestellten wert 
Anders verhält es sich wenn der smart-micro-solar mit im spiel ist 
DTU neustart geht auf 0%  der micro geht auch auf 0 teilweise auch auf -1 und der WR wird abgeregelt auf 0 ja auf 0 % und der WR schaltet nicht ab Hmm ???? und auch nach stunden ändert sich nichts alles auf 0% 
endweder ist die DTU vergesslich geworden oder fehler liegt wo anders ........ 
wenn ich temp 1% eingebe und den WR neu starte springt er zeitnah auf seinen eingetellten wert (zb 30%)  bei 0% macht er das nicht

2x hm 1500

Gruß aus Konz
